### PR TITLE
ログイン機能の完成

### PR DESCRIPTION
**Pull request by igamasayuki**
ログイン機能を完成させましたので、レビューをお願いします。

### CODE REVIEW

File: src/main/java/com/example/controller/AdministratorController.java [hunk 1]
```diff
@@ -57,6 +57,7 @@ public String insert(InsertAdministratorForm form) {
 		// フォームからドメインにプロパティ値をコピー
 		BeanUtils.copyProperties(form, administrator);
 		administratorService.insert(administrator);
+
```
Comment (igamasayuki) [inline src/main/java/com/example/controller/AdministratorController.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/EmployeeController.java [hunk 2]
```diff
@@ -57,7 +57,7 @@ public String showList(Model model) {
 	 */
 	@GetMapping("/showDetail")
 	public String showDetail(String id, Model model, UpdateEmployeeForm form) {
-		Employee employee = employeeService.showDetail(Integer.parseInt(id));
+			Employee employee = employeeService.showDetail(Integer.parseInt(id));
```
Comment (igamasayuki) [inline src/main/java/com/example/controller/EmployeeController.java:60]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (igamasayuki) [inline src/main/java/com/example/controller/EmployeeController.java:60]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 24, expected level should be 4.

File: src/main/java/com/example/controller/EmployeeController.java [hunk 3]
```diff
@@ -73,7 +73,8 @@ public String showDetail(String id, Model model, UpdateEmployeeForm form) {
 	 */
 	@PostMapping("/update")
 	public String update(@Validated UpdateEmployeeForm form, BindingResult result, Model model) {
-		if (result.hasErrors()) {
+		if (result.hasErrors())
```
Comment (igamasayuki) [inline src/main/java/com/example/controller/EmployeeController.java:76]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (igamasayuki) [inline src/main/java/com/example/controller/EmployeeController.java:76]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/EmployeeController.java [hunk 4]
```diff
@@ -73,7 +73,8 @@ public String showDetail(String id, Model model, UpdateEmployeeForm form) {
 	 */
 	@PostMapping("/update")
 	public String update(@Validated UpdateEmployeeForm form, BindingResult result, Model model) {
-		if (result.hasErrors()) {
+		if (result.hasErrors()) 
+		{
```
Comment (igamasayuki) [inline src/main/java/com/example/controller/EmployeeController.java:77]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (igamasayuki) [inline src/main/java/com/example/controller/EmployeeController.java:77]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.blocks.LeftCurlyCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'{' at column 3 should be on the previous line.

Comment (igamasayuki) [inline src/main/java/com/example/controller/EmployeeController.java:77]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if lcurly' has incorrect indentation level 16, expected level should be 6.

**Review by runy-iga1 — APPROVED** [review]
承認します。

**Review by runy-iga1 — APPROVED** [review]
承認します。

**Review by runy-iga1 — APPROVED** [review]
OK

**Pull request closed without merging**